src/annotator/integrations/test/pdf-test.js

```diff
@@ -104,6 +104,7 @@ describe('annotator/integrations/pdf', () => {
       pdfIntegration?.destroy();
       delete window.PDFViewerApplication;
       outerContainer.remove();
+      document.querySelector('hypothesis-banner')?.remove(); // <hypothesis-banner> elements are created outside the outerContainer
```

Comment:
Since the `PDFIntegration` instance creates the banner, it should probably destroy it too in the `destroy` method.

Reply:
That's a good suggestion, I have added a second commit for that.